### PR TITLE
use mirrored sources for some plugins

### LINKED ISSUE #23: Use stable links
Currently the links seem to be changing here. We need to pick something that is static

https://github.com/conda-forge/nsis-feedstock/pull/20#discussion_r1085400238

https://github.com/conda-forge/nsis-feedstock/pull/20#discussion_r1085400395

cc @isuruf

### PR DESCRIPTION
**Pull request by jaimergp**
Checklist
* [X] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [X] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [ ] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [X] Ensured the license file is being packaged.

Closes https://github.com/conda-forge/nsis-feedstock/issues/23

### CODE REVIEW

**Comment by jaimergp** [comment]
@conda-forge-admin, please rerender

**Comment by jaimergp** [comment]
@conda-forge-admin, please rerender

**Comment by jaimergp** [comment]
@jakirkham @isuruf - this is ready for review! Thanks 🙏

**Review by isuruf — COMMENTED** [review]
Do you want to update nsis-untgz and nsis-UnicodePathTest as well?

File: recipe/meta.yaml [hunk 1]
```diff
@@ -34,14 +34,12 @@ source:
   - url: https://github.com/mingwandroid/nsis-UnicodePathTest/archive/fa74caef553883f1820049d89e169aff57551796.zip
```
Comment (jaimergp) [inline recipe/meta.yaml]:
```suggestion
  - url: https://github.com/conda/nsis-mirror/releases/download/v308/nsis-UnicodePathTest-fa74caef553883f1820049d89e169aff57551796.zip
```

File: recipe/meta.yaml [hunk 2]
```diff
@@ -19,13 +19,13 @@ source:
...
     folder: plugins/BgWorker
-  - url: https://code.kliu.org/misc/elevate/elevate-1.3.0-redist.7z
+  - url: https://github.com/conda/nsis-mirror/releases/download/v308/elevate-1.3.0-redist.7z
     sha256: b1b3f070353a0eadee2cea3a575049d10df9763ff24e39313da4cec9455382e1
     folder: plugins/elevate
-  - url: https://nsis.sourceforge.io/mediawiki/images/8/8f/UAC.zip
+  - url: https://github.com/conda/nsis-mirror/releases/download/v308/UAC-0.2.4c-20150526.zip
     sha256: 20e3192af5598568887c16d88de59a52c2ce4a26e42c5fb8bee8105dcbbd1760
     folder: plugins/UAC
   - url: https://github.com/mingwandroid/nsis-untgz/archive/5c814c5f2c8a9a14e3a6ddd3e594fcc76db5b86a.zip
```
Comment (jaimergp) [inline recipe/meta.yaml]:
```suggestion
  - url: https://github.com/conda/nsis-mirror/releases/download/v308/nsis-untgz-5c814c5f2c8a9a14e3a6ddd3e594fcc76db5b86a.zip
```

**Comment by jaimergp** [comment]
Done @isuruf

**Comment by isuruf** [comment]
Thanks

**Comment by jakirkham** [comment]
Thank you both! 🙏

**Pull request merged**